internal/pipe/scoop/scoop_test.go

```diff
@@ -1051,6 +1059,23 @@ func TestRunPipeScoopWithSkip(t *testing.T) {
 	require.NoError(t, err, "file should exist: "+distFile)
 }
 
+func TestRunPipeScoopWithSkipPublish(t *testing.T) {
+	folder := t.TempDir()
+	ctx, path := getScoopPipeSkipCtx(folder)
+	ctx.SkipPublish = true
+
+	f, err := os.Create(path)
+	require.NoError(t, err)
+	require.NoError(t, f.Close())
+
+	cli := &DummyClient{}
+	require.EqualError(t, doRun(ctx, cli), pipe.ErrSkipPublishEnabled.Error())
+
+	distFile := filepath.Join(folder, ctx.Config.Scoop.Name+".json")
+	_, err = os.Stat(distFile)
+	require.NoError(t, err, "file should exist: "+distFile)
```

Comment:
can use [`require.FileExists`](https://pkg.go.dev/github.com/stretchr/testify@v1.7.0/require#FileExists)

```suggestion
	require. FileExists(t, distFile)
```